.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "grb2.h": "c"
```

Comment:
Is this file intentional?

Reply:
Nope, I'll remove it. Thanks for pointing it out.